.dockerignore

```diff
@@ -0,0 +1,7 @@
+/target/
+/.git/
+.github/
+.gitignore
+/tests/
+README.md
+/example-files/
```

Comment:
I updated the docker ignore to remove all slashes. Not necessary because it matches glob patterns.

See this example: https://github.com/kpcyrd/mini-docker-rust/blob/main/.dockerignore

Reply:
Fixed.